### PR TITLE
Updated db_handler.py

### PR DESCRIPTION
**Pull request by mukhtarB**
- Added CRUD functionality for accessing the db.
- Converted read query to a more generic read.
- It uses the old plugin ID for dms (since we dont have a combined plugin ID yet. 
     Effect -> all data will be saved on dm collections)
- ORG_ID is also the one we've been using before
- Centrifugo token is older one (which i believe was specific for dms only also)
- login_user() is added temporarily, but it needs to go in the long run and become more dynamic

### CODE REVIEW

File: backend/utils/db_handler.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+            response = requests.put(url=self.write_api, json=body)
+        except requests.exceptions.RequestException as e:
+            print(e)
+            return None
+        if response.status_code == 200:
+            return response.json()
+        else:
+            return {"status_code": response.status_code, "message": response.reason}
+
+    # async def read(self, collection_name, filter={}):
```
Comment (zxenonx) [inline backend/utils/db_handler.py]: We can remove that part as it has been refactored below

File: backend/utils/db_handler.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        except requests.exceptions.RequestException as e:
+            print(e)
+            return None
+        if response.status_code == 200:
+            return response.json().get("data")
+        else:
+            return {"status_code": response.status_code, "message": response.reason}
+
+    async def delete(self, collection_name, document_id):
+    """
```
Comment (zxenonx) [inline backend/utils/db_handler.py:139]: For the docstrings, i want us to stick with Google doctrings format:

"""Gets and prints the spreadsheet's header columns

Args:
    file_loc (str): The file location of the spreadsheet
    print_cols (bool): A flag used to print the columns to the console
        (default is False)

Returns:
    list: a list of strings representing the header columns
"""
From the Google Python Style Guide: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings

**Review by zxenonx — CHANGES_REQUESTED** [review]

File: backend/utils/db_handler.py (hunk 1 above)
Comment (zxenonx) [reply backend/utils/db_handler.py]: @mukhtarB

File: backend/utils/db_handler.py (hunk 2 above)
Comment (zxenonx) [reply backend/utils/db_handler.py:139]: @mukhtarB

File: backend/utils/db_handler.py (hunk 1 above)
Comment (mukhtarB) [reply backend/utils/db_handler.py]: Oh you mean the commented out code right... just left it incase I had to use it later... I'm definitely taking it out

File: backend/utils/db_handler.py (hunk 2 above)
Comment (mukhtarB) [reply backend/utils/db_handler.py:139]: Okay so args and returns, as opposed to params and returns.... 
Also, do you want to add a raises field @zxenonx

File: backend/utils/db_handler.py (hunk 1 above)
Comment (mukhtarB) [reply backend/utils/db_handler.py]: @zxenonx

Comment (zxenonx) [reply backend/utils/db_handler.py]: Yeah, the commented out code. OK, please remove it ASAP. 👍

File: backend/utils/db_handler.py (hunk 2 above)
Comment (zxenonx) [reply backend/utils/db_handler.py:139]: Only if the function raises an exception

File: backend/utils/db_handler.py (hunk 1 above)
Comment (mukhtarB) [reply backend/utils/db_handler.py]: done

File: backend/utils/db_handler.py (hunk 2 above)
Comment (mukhtarB) [reply backend/utils/db_handler.py:139]: done

**Review by zxenonx — APPROVED** [review]

**Review by o4codes — APPROVED** [review]

**Pull request merged**